nov-13-hardfork-spec.md

```diff
@@ -0,0 +1,38 @@
+# November 13th Bitcoin Cash Hardfork Technical Details
+
+Version 1.0, 2017-12-01
+
+## Summary
+ 
+When the median time past[1] of the most recent 11 blocks (MTP-11) is greater than or equal to UNIX timestamp 1510600000 Bitcoin Cash will execute a hardfork according to this specification.  It includes the following three protocol changes:
+
+* Enforcement of LOW_S signatures (BIP 0146)
```

Comment:
Maybe provide direct link references to

- https://github.com/bitcoin/bips/blob/master/bip-0146.mediawiki#low_s
- https://github.com/bitcoin/bips/blob/master/bip-0146.mediawiki#nullfail